src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -225,10 +225,18 @@ private static LibraryDependency ToPackageLibraryDependency(PackageReference ref
 
         private static VersionRange ToVersionRange(string version, bool isCpvmEnabled)
         {
-            if (isCpvmEnabled && string.IsNullOrEmpty(version))
+            if (string.IsNullOrEmpty(version))
             {
-                // Projects that have their packages managed centrally will not have Version metadata on PackageReference items.
-                return null;
+                if (isCpvmEnabled)
+                {
+                    // Projects that have their packages managed centrally will not have Version metadata on PackageReference items.
+                    return null;
+                }
+                else
+                {
+                    // Below is equalant as version doesn't have an inclusive lower bound.
```

Comment:
I don't think this adds much value. 
It contains traces about the issue, rather than the implementation itself.

I'd remove the comment altogether.

Reply:
Ok. Removed.